### PR TITLE
change tests to not always pass

### PR DESCRIPTION
**Pull request by dmt**
https://github.com/Microsoft/TypeScript-Node-Starter/issues/37

As mentioned in the issue, I don't really know what I'm doing but this way I can get tests to fail if I change expectations/expected behaviour.

### CODE REVIEW

**Comment by Strnadj** [comment]
Please @dmt could you explain why you're using URI other than express app instance? According to supertest documentation you can `request(app)`. I am just curious.

**Comment by dmt** [comment]
@Strnadj I didn't write those tests, I only changed the port. 

I guess that if `server.ts` wasn't binding to the the port, you could just require it and use it for supertest but I didn't want to rewrite an app I really don't know anything about.

**Comment by markmssd** [comment]
+1, I think this is a very important PR that should be merged :/

**Comment by dmt** [comment]
Whatever happened to this?

File: test/api.test.ts [hunk 1]
```diff
@@ -1,11 +1,12 @@
-import {} from 'jest';
+import {} from "jest";
```
Comment (felangel) [inline test/api.test.ts]: Do we need this import? Seems like it's not being used anywhere?

**Comment by bowdenk7** [comment]
hey @dmt - sorry I was away for a while.. but I'm back and going forward committed to keeping this repo fresh. I definitely want to take these changes. There are now merge conflicts from some other PRs I merged, so if you want to update this PR that'd be great. Otherwise I'll resolve them sometime this evening or tomorrow.

Apologies again for the long delay!

File: test/api.test.ts (hunk 1 above)
Comment (dmt) [reply test/api.test.ts]: I tried to change as little as possible. The quote change was just following the codestyle, IIRC.

**Comment by dmt** [comment]
@bowdenk7 Thanks. I struggle a little remembering all the context. In particular the changes to server.ts while the contents moved to app.ts look like something I can't resolve without spending a bit more time getting back into it. So if you can resolve this, that would be appreciated.

**Pull request merged**